internal/controlplane/handlers_repositories.go

```diff
@@ -98,18 +98,13 @@ func (s *Server) ListRepositories(ctx context.Context,
 	entityCtx := engine.EntityFromContext(ctx)
 	projectID := entityCtx.Project.ID
 
-	provider, err := getProviderFromRequestOrDefault(ctx, s.store, in, projectID)
-	if err != nil {
-		return nil, providerError(err)
-	}
-
```

Comment:
This is removing the ability to list repositories by provider.  Intentional?

Reply:
(If repos are associated with providers, it seems useful to be able to limit the listo to one provider, to answer "what happens if I delete this" if nothing else.)

Reply:
Yeah I can see us wanting to do that in the CLI. I'll update this to use the provider if it's in the context.